app/views/docs/command-line.phtml

```diff
@@ -20,6 +20,14 @@
 <ul class="phases clear" data-ui-phases>
     <li>
         <h4>MacOS</h4>
+        <p>Using <a href="https://brew.sh" target="_blank">Homebrew</a></p>
+        <div class="ide margin-bottom" data-lang="bash" data-lang-label="Bash">
+            <pre class="line-numbers"><code class="prism language-bash" data-prism>$ brew tap appwrite/sdk-for-node-cli https://github.com/appwrite/sdk-for-node-cli
```

Comment:
At what point can we drop the repo link part? if any?

Reply:
We have 3 requirements
1. Minimum 30 Github stars 
2. Minimum x forks
3. A community member needs to submit our package for being discoverable